packages/cli/lib/artifact-generator.js

```diff
@@ -94,18 +94,33 @@ module.exports = class ArtifactGenerator extends BaseGenerator {
        * Both those properties must be present for this to happen. Optionally,
        * this can be disabled even if the properties are present by setting:
        * this.artifactInfo.disableIndexUpdate = true;
+       *
+       * Multiple indexes / files can be updated by providing an array of
+       * index update objects as follows:
+       * this.artifactInfo.updateIndexes = [{
+       *   dir: 'directory in which to update index.ts',
+       *   file: 'file to add to index.ts',
+       * }, {dir: '...', file: '...'}]
        */
-      if (
-        this.artifactInfo.updateIndexes &&
-        !this.artifactInfo.disableIndexUpdate
-      ) {
-        for (let i = 0; i < this.artifactInfo.updateIndexes.length; i++) {
-          const update = this.artifactInfo.updateIndexes[i];
-          await updateIndex(update.dir, update.file);
+      if (!this.artifactInfo.disableIndexUpdate) {
+        if (
+          !this.artifactInfo.updateIndexes &&
```

Comment:
We should change the name `updateIndexes` to something like `indexesToBeUpdated` since it could be functionally confusing with `disableIndexUpdate`